### ISSUE TITLE
`cy.readFile()` in `example_spec.js` fails on non-existing file

### ISSUE THREAD

**Issue by ThaNarie** [cypress-io/cypress#414]
**Description**
When running the `example_spec.js` after adding a project in the GUI, the `cy.readFile()` tests fails on a file that is not added by 'adding the project'.

**Steps To Reproduce**
- [ ] Add a project to the GUI tool
- [ ] Click the `example_spec.js` test to run the just added tests
- [ ] See the `cy.readFile()` fail.

**Additional Info**
https://www.dropbox.com/s/8uye4kx0owbrnia/Screenshot%202017-02-06%2022.21.20.png?dl=0

```
Error:      CypressError: Timed out retrying: cy.readFile("app/commands/actions.html") failed because the file does not exist at the following path:

/Volumes/Projects/.../project-root/app/commands/actions.html
```

https://github.com/cypress-io/cypress-example-kitchensink/blob/1c9ba60a9ec2d1e0486eae738ebd857d73f240da/cypress/integration/example_spec.js#L1111

**Comment by jennifer-mann** [comment]
[PR to fix this issue](https://github.com/cypress-io/cypress-example-kitchensink/pull/7). Issue will be closed when seeded `example_spec.js` file is updated in next Cypress release.

**Comment by brian-mann** [comment]
Fixed in [`0.19.1`](https://github.com/cypress-io/cypress/wiki/Changelog#0191-03092017).
